server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -64,6 +66,7 @@ public NodesStatsRequest(StreamInput in) throws IOException {
         indices = new CommonStatsFlags(in);
         requestedMetrics.clear();
         requestedMetrics.addAll(in.readStringList());
+        restActionsFilters.addAll(in.readStringList());
```

Comment:
`./gradlew ':qa:mixed-cluster:v2.5.0#mixedClusterTest'` task would be failing due to missing backward-compatibility check here too.

I would suggest doing the following:
1. Raise the initial PR to 'main' branch with checks against `Version.V_3_0_0`.
2. Raise the backport PRs to 2.x and 2.5 branches with check against `Version.V_2_5_0`.
3. Raise a follow up PR to 'main' branch with checks against `Version.V_2_5_0`.

Reply:
+1 to @ketanv3 suggestion